modules/scalacheck/shared/src/test/scala/weaver/scalacheck/PropertyDogFoodTest.scala

```diff
@@ -64,15 +64,19 @@ object PropertyDogFoodTest extends IOSuite {
 
   test("Config can be overridden") { dogfood =>
     expectErrorMessage(
-      s"Discarded more inputs (${Meta.ConfigOverrideChecks.configOverride.maximumDiscarded}) than allowed",
+      s"Discarded more inputs (${Meta.ConfigOverrideChecks.configOverride.maximumDiscarded + 1}) than allowed",
       dogfood.runSuite(Meta.ConfigOverrideChecks))
   }
 
   test("Discarded counts should be accurate") { dogfood =>
     expectErrorMessage(
-      s"Discarded more inputs (${Meta.DiscardedChecks.checkConfig.maximumDiscarded}) than allowed",
+      s"Discarded more inputs (${Meta.DiscardedChecks.checkConfig.maximumDiscarded + 1}) than allowed",
```

Comment:
I'd be surprised to see a number different than what's in the config. How about a slightly redacted message?
```suggestion
      s"Discarded more inputs than allowed. Input discard limit is ${Meta.DiscardedChecks.checkConfig.maximumDiscarded}.",
```

Reply:
Definitely! I've gone with the slightly more terse:
```
Discarded more inputs (2) than allowed (1)
```